src/app/core/directives/not-feature-toggle.directive.ts

```diff
@@ -16,20 +18,43 @@ import { FeatureToggleService } from 'ish-core/utils/feature-toggle/feature-togg
 @Directive({
   selector: '[ishNotFeature]',
 })
-export class NotFeatureToggleDirective {
+export class NotFeatureToggleDirective implements OnDestroy {
+  private subscription: Subscription;
+  private enabled$ = new ReplaySubject<boolean>(1);
+
+  private destroy$ = new Subject();
+
   constructor(
     private templateRef: TemplateRef<unknown>,
     private viewContainer: ViewContainerRef,
-    private featureToggle: FeatureToggleService
-  ) {}
+    private featureToggle: FeatureToggleService,
+    private cdRef: ChangeDetectorRef
+  ) {
+    this.enabled$.pipe(distinctUntilChanged(), takeUntil(this.destroy$)).subscribe(enabled => {
+      if (enabled) {
```

Comment:
I think it's a little confusing that you check for enabled here when you actually pass `!value` to `enabled$`. 
So either this variable should be called `disabled` or you could pass the normal value to `enabled$` and check for the negation here, in my opinion.
Of course feel free to give me your thoughts on this or correct me if I misunderstood something.

Reply:
Agreed. Applied the first suggested change.

Reply:
Ok glad that you agree @cesarGamaSa .
Sorry to be so picky, but now the `enabled$` Subject is carrying a value called `disabled`. It doesn't really make sense and still makes it confusing for me. 
It seems more logical that as soon as you negate the result from `featureService.enabled$` the stream and values should be called disabled.

Reply:
Done :)